stats/record.go

```diff
@@ -24,20 +24,8 @@ import (
 
 // Record records one or multiple measurements with the same tags at once.
 // If there are any tags in the context, measurements will be tagged with them.
-//
-// Record returns fast and recorded measurements will have little overhead
-// if there are no views assicaiated with them.
 func Record(ctx context.Context, ms ...Measurement) {
 	if internal.DefaultRecorder != nil {
-		var record bool
```

Comment:
Why are we removing this code? It doesn't hurt to skip recording if there is no Measure attached,
plus this seems to revert https://github.com/census-instrumentation/opencensus-go/pull/468 whose purpose was to improve performance

Reply:
and in particular https://github.com/census-instrumentation/opencensus-go/pull/468/files#diff-8f16a4ad787d3d7d83413c0f7415e357

Reply:
Nothing ever set measure.views so this code path was never taken